spring-cloud-gcp-data-firestore/src/main/java/org/springframework/cloud/gcp/data/firestore/util/Util.java

```diff
@@ -27,7 +25,20 @@ private Util() {
 	}
 
 	public static String extractDatabasePath(String parent) {
-		return parent.substring(0, StringUtils.ordinalIndexOf(parent, "/", 4));
```

Comment:
What happened to `StringUtils`? Why can't we use it anymore?

Reply:
`org.apache.commons.lang3` was removed from dependencies and I don't want to bring it in only for this one method.